### PR TITLE
define BEP on first usage

### PR DESCRIPTION
**Pull request by seb-1204**

### CODE REVIEW

**Comment by kluppy** [comment]
@st-review merge

**Comment by st-review** [comment]
@kluppy: Couldn't retrieve your user information - not merging.

File: specs/bep-v1.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Block Exchange Protocol v1
 Introduction and Definitions
 ----------------------------
 
-BEP is used between two or more *devices* thus forming a *cluster*. Each
+Block Exchange Protocol (BE0)P is used between two or more *devices* thus forming a *cluster*. Each
```
Comment (calmh) [inline specs/bep-v1.rst]: Typo `BE0)P`

**Comment by calmh** [comment]
Added manually, thanks for the suggestion!

**Pull request closed without merging**